convert2rhel/systeminfo.py

```diff
@@ -359,6 +364,80 @@ def corresponds_to_rhel_eus_release(self):
         """
         return self.releasever in EUS_MINOR_VERSIONS
 
+    def _is_dbus_running(self):
+        """
+        Check whether dbus is running.
+
+        :returns: True if dbus is running.  Otherwise False
+        """
+        retries = 0
+        status = False
+
+        while retries < CHECK_DBUS_STATUS_RETRIES:
+            if self.version.major <= 6:
+                status = _is_sysv_managed_dbus_running()
+            else:
+                status = _is_systemd_managed_dbus_running()
+
+            if status is not None:
+                # We know that DBus is definitely running or stopped
+                break
+
+            # Wait for 1 second, 2 seconds, and then 4 seconds for dbus to be running
+            # (In case it was started before convert2rhel but it is slow to start)
+            time.sleep(2**retries)
+            retries += 1
+
+        else:  # while-else
+            # If we haven't gotten a definite yes or no but we've exceeded or retries,
+            # report that DBus is not running
+            status = False
+
+        return status
+
+
+def _is_sysv_managed_dbus_running():
+    """Get DBus status from SysVinit compatible systems."""
+    # None means the status should be retried because we weren't sure if it is turned off.
+    running = None
+    output, _ret_code = utils.run_subprocess(["/sbin/service", "messagebus", "status"])
```

Comment:
`FileNotFoundError: [Errno 2] No such file or directory: '/sbin/service': '/sbin/service'` on CentOS Linux 8.

Reply:
I think I've tracked this down to the pretend_os fixture which populates system_info.  Inside of a container we might not have an init system so we need to hardcode a value for dbus_running rather than trying to use `service` or `systemctl` to probe for it.  Should be fixed now.